src/modules/account/components/account-legacy-rep/account-legacy-rep.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { augur } from 'services/augurjs'
+
+import Styles from 'modules/account/components/account-legacy-rep/account-legacy-rep.styles'
+
+export default class AccountLegacyRep extends Component {
```

Comment:
Can this be a stateless component?

Reply:
maybe? would the format for the functions provided in the props still function if they were just referenced directly? I imagine so but I'm depending on a lot of magic boilerplate everywhere so I'm not sure.

Reply:
Yup, seems fine! Will change